deploy/registration-service.yaml

```diff
@@ -104,26 +112,67 @@ objects:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                - name: REGISTRATION_ENVIRONMENT
+                - name: REGISTRATION_SERVICE_CONFIG_MAP_NAME
+                  value: ${CONFIG_MAP_NAME}
```

Comment:
@MatousJobanek i think it may be better to have it as a parameter so that whoever creates the configmap can configure this themselves - this isn't a configmpa that we create. it will be documented in toolchain-infra though..

Reply:
ok, so you create a logic that loads another ConfigMap with values that can override those values set in the default ConfigMap `registration-service`...? This is well... ...unexpected. 
Wouldn't it be better to incorporate it into the host-operator CM and update the default `registration-service` CM through the RegistrationService_controller?

In addition, what I'm missing is the scenario of setting/changing the `REGISTRATION_SERVICE_CONFIG_MAP_NAME` value. Don't forget that this deployment is managed by the controller.

Reply:
> the registration_service_config_map_name can be set in the env files. 

but who will set it? If it is set in the runtime (when the reg-service is deployed and runnnig), then it will be replaced to the original value.

> I'm not too sure why it's unexpected?

because we set everything through the host-operator, so I'm not sure why this is separated. We already have a ConfigMap for reg-service, so I just didn't expect another one. 

>  it allows us to create configmaps and secrets the same way we do for host-operator and member-operator.

yeah, so why don't use the configmap/secret of the host-operator? The fact that registration service is a another service/deployment is just a implementation detail of the host-operator.

Reply:
@MatousJobanek after a call with @alexeykazakov  we have agreed to remove the configmap stuff and for now hardcodde the name of the secret in configuration.go